### PR TITLE
(vuepress-rewrite) Some WP notes + WIP reverting to stock catagory

### PR DESCRIPTION
**Pull request by CoolPenguin27**
This pr does the following:
 -  Add mention that you don't need to disable wp if only flashing RW_Legacy,
 -  Adds flash failure note in flashing-firmware.md.
 -  WIP reverting to stock rewrite.

### CODE REVIEW

**Comment by ethanaobrien** [comment]
Can we put the reverting directory in the docs folder

File: src/docs/firmware/flashing-firmware.md [hunk 1]
```diff
@@ -14,13 +14,17 @@ Refer to [this page](about) for an overview of both RW_LEGACY and UEFI.
...
 
 1. [Enable developer mode](developer-mode).
-2. [Disable write protect](write-protect).
+2. [Disable write protect](write-protect) (If flashing UEFI).
 3. Run [MrChromebox's firmware utility script](https://mrchromebox.tech/#fwscript) within VT-2 (`ctrl` + `alt` + `f2` (right arrow)).
     * Type `cd; curl -LO mrchromebox.tech/firmware-util.sh && sudo bash firmware-util.sh` and press Enter.
     * If you encounter certificate related errors when downloading the script from ChromeOS, then add `-k` to the `curl` and script command to bypass SSL certificate checking as so:
         * `cd; curl -LOk mrchromebox.tech/firmware-util.sh && sudo bash firmware-util.sh`
 4. If you are flashing Full ROM, the firmware script will prompt you to make a backup of your stock firmware. **Store it in a safe place** (Google Drive, another PC, etc), as reverting to stock firmware without a backup is **very difficult**.
-5. Shutdown the device.
```
Comment (ethanaobrien) [inline src/docs/firmware/flashing-firmware.md:23]: I think we should still have a step 5 here, maybe something like `5. If there is no error, shutdown the device without worry`. Repetitiveness of something important isn't bad (to an extent)

File: src/reverting/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Requirements
+
+- A PC or laptop to make a ChromeOS recovery USB.
+- An 8GB or larger USB drive.
+- An understanding that this guide has the potential to brick your device.
+- Your stock ROM backup.
+
+You can revert your firmware back to stock and reinstall chromeOS.
+
+This is usefull if your planning to sell your device.
```
Comment (ethanaobrien) [inline src/reverting/index.md:23]: you're

File: src/reverting/installing.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Installing chromeOS
+
+::: tip
+If recovery fails, unplug the battery since recovery also attempts to update the firmware.
```
Comment (ethanaobrien) [inline src/reverting/installing.md:4]: Could also be jumper, whatever you did last time to disable wp

File: src/reverting/installing.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+:::
+
+**Booting the Recovery USB**
+1. Enter [recovery mode](recovery-mode.html).
+2. Plug in your recovery USB drive
+3. Begin the recovery process
+
+## Making a Recovery USB
+
+1. Download the [Chromebook Recovery Utility](https://chrome.google.com/webstore/detail/chromebook-recovery-utili/pocpnlppkickgojjlmhdmidojbmbodfm) from the Chrome Web Store
```
Comment (ethanaobrien) [inline src/reverting/installing.md:14]: Maybe add a from linux section (download using cros.tech and burn with dd)?

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

**Comment by CoolPenguin27** [comment]
> Can we put the reverting directory in the docs folder

Yeah I misplaced without realizing

**Comment by CoolPenguin27** [comment]
yeah it might be better to split "Making a Recovery USB" to its own page

File: src/reverting/installing.md (hunk 3 above)
Comment (CoolPenguin27) [reply src/reverting/installing.md]: do you know other people in the disc that needed to undo jumper/ccd to flash recovery?

Comment (ethanaobrien) [reply src/reverting/installing.md]: It's just that you need to disable hardware write protect, not all devices disable hardware write protect through battery

File: src/docs/reverting/making-recovery-usb.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## Manually Flash Recovery Image (Windows/ChromeOS/MacOS/Linux)
+
+1. Go to [cros.tech](https://cros.tech/) and find your device.
+
+2. Select on "Recovery Images" and select the latest one.
+
+3. Extract the file.
+
+**Windows**
```
Comment (ethanaobrien) [inline src/docs/reverting/making-recovery-usb.md:31]: Can we use `### Windows` (or `#### Windows`) for this section and linux and macOS/chromeos too

(macOS is spelled with a lowercase m)

File: src/docs/firmware/supported-devices.md [hunk 6]
```diff
@@ -2231,4 +2231,5 @@ If on on a smaller screen, scroll sideways to see whole table.
         </tr>
     </tbody>
 </table>
+
```
Comment (ethanaobrien) [inline src/docs/firmware/supported-devices.md:2234]: ?

File: src/.vuepress/config.js [hunk 7]
```diff
@@ -190,9 +190,28 @@ export default {
           {
             text: 'Linux distro-specific Configuration',
             link: '/docs/advanced/distros',
-          }
+          },
```
Comment (ethanaobrien) [inline src/.vuepress/config.js:193]: Shouldn't be a comma here

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: src/docs/firmware/supported-devices.md (hunk 6 above)
Comment (CoolPenguin27) [reply src/docs/firmware/supported-devices.md:2234]: I think it was an accident that came up when trying to fix merge conflicts earlier

**Pull request merged**